sandbox/rocky/tf/plotter/plotter.py

```diff
@@ -0,0 +1,90 @@
+import atexit
+from multiprocessing import Process
+import numpy as np
+import pickle
+import platform
+from queue import Empty, Queue
+import tensorflow as tf
+from threading import Thread
+
+from rllab.sampler.utils import rollout
+
+__all__ = ['init_worker', 'init_plot', 'update_plot']
+
+process = None
+queue = None
+sess = None
+
+
+def _worker_start():
+    env = None
+    policy = None
+    max_length = None
+    global sess
+    try:
+        with sess.as_default():
+            with sess.graph.as_default():
+                while True:
+                    msgs = {}
+                    # Only fetch the last message of each type
+                    while True:
+                        try:
+                            msg = queue.get_nowait()
```

Comment:
This generates busy waiting if there's nothing in the queue.

Reply:
This should be fine for now because something will be placed into the queue for each iteration, and upon completion the thread will close.